maintainer/maintainer.py

```diff
@@ -1,31 +1,17 @@
 import logging
-
-import ts3
-
 import configuration
+from time import sleep
 from channels import ChannelsMaintanance
 
-logging.basicConfig(level=logging.DEBUG)
-
-
-
-def establish_connection():
-    ts3conn = ts3.query.TS3Connection(configuration.ip, configuration.port)
-    ts3conn.login(client_login_name=configuration.client_login_name, client_login_password=configuration.client_login_password)
-    ts3conn.use(sid=configuration.sid)
-    ts3conn.clientupdate(client_nickname="Huskar")
-#    ts3conn.keepalive(10)
-    return ts3conn
-
-
-
 
 def main():
-    ts3conn = establish_connection()
-    x = ChannelsMaintanance(ts3conn, 7, 7)
-    x.check_channels()
+    maintenance = ChannelsMaintanance(7)
+    while True:
+        maintenance.channels_to_quarantine()
+        maintenance.delete_children()
+        maintenance.delete_parents()
+        time.sleep(180)
```

Comment:
I think this might be improved, rethink if you want to have such code as a production code.